trillian/testdata/Makefile

```diff
@@ -1,7 +1,7 @@
 all: pubkeys ca leaves
 
 # The following private keys are never regenerated.
-SERVER_PRIVKEYS=ct-http-server.privkey.pem log-rpc-server.privkey.pem map-rpc-server.privkey.pem
+SERVER_PRIVKEYS=ct-http-server.privkey.pem log-rpc-server.privkey.pem
```

Comment:
Seems unrelated?

Reply:
Yeah, it is a bit -- this is (I think) the first time any of the testdata has been rebuilt since the repo split, and the `Makefile` didn't get fully updated at the time.  I'll pull it out into a separate commit